hedera-node/hedera-consensus-service-impl/src/test/java/module-info.java

```diff
@@ -2,7 +2,15 @@
     requires com.hedera.node.app.service.consensus;
     requires com.hedera.node.app.service.consensus.impl;
     requires org.junit.jupiter.api;
+    requires com.hedera.node.app.service.mono.testFixtures;
+    requires com.hedera.node.app.service.mono;
+    requires org.mockito;
+    requires org.mockito.junit.jupiter;
+    requires org.assertj.core;
+    requires com.google.protobuf;
 
     opens com.hedera.node.app.service.consensus.impl.test to
             org.junit.platform.commons;
+    opens com.hedera.node.app.service.consensus.impl.handlers.test to
```

Comment:
for tests we can just open the complete model:

`open model x.y {}`

By doing so no opens need to be specified for packages.